lib/lltz_ir/dsl.ml

```diff
@@ -0,0 +1,183 @@
+(* dsl.ml *)
+open Grace
+module LE = Expr
+module LT = Type
+module LP = Primitive
```

Comment:
Module names are Snake_case.
Suggest 
```ocaml
module E = Expr
module T = Type
module P = Primitive
```

Reply:
the idea here is to be consistent with the compilation where L-prefixed modules are related to LLTZ and M-prefixed modules are related to Michelson. I chose LE as acronym for LLTZ Expression, which I think should be fine in snake case. ALternatives are e.g. L_e, which seems a bit ugly to me.

Reply:
Why not go for
> module LLTZ = struct
>   module E = Expr
>   module T = Type
>   module P = Primitive
> end 
and use LLTZ.E etc.?

Reply:
yeah, I think this works, will change it like this